[2024-11-21T00:15:08.748+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-21T00:15:08.759+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_heart_disease_data.split_data manual__2024-11-21T00:15:06.390298+00:00 [queued]>
[2024-11-21T00:15:08.765+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_heart_disease_data.split_data manual__2024-11-21T00:15:06.390298+00:00 [queued]>
[2024-11-21T00:15:08.766+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-21T00:15:08.774+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): split_data> on 2024-11-21 00:15:06.390298+00:00
[2024-11-21T00:15:08.783+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3434) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-21T00:15:08.783+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'process_heart_disease_data', 'split_data', 'manual__2024-11-21T00:15:06.390298+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpwgpwlvii']
[2024-11-21T00:15:08.784+0000] {standard_task_runner.py:72} INFO - Started process 3447 to run task
[2024-11-21T00:15:08.784+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask split_data
[2024-11-21T00:15:08.818+0000] {task_command.py:467} INFO - Running <TaskInstance: process_heart_disease_data.split_data manual__2024-11-21T00:15:06.390298+00:00 [running]> on host 2276f79c0942
[2024-11-21T00:15:08.866+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_heart_disease_data' AIRFLOW_CTX_TASK_ID='split_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-21T00:15:06.390298+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-21T00:15:06.390298+00:00'
[2024-11-21T00:15:08.867+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-21T00:15:08.912+0000] {logging_mixin.py:190} INFO - Dane podzielone na 'modelowy.csv' i 'douczeniowy.csv'
[2024-11-21T00:15:08.913+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-21T00:15:08.920+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-21T00:15:08.920+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=process_heart_disease_data, task_id=split_data, run_id=manual__2024-11-21T00:15:06.390298+00:00, execution_date=20241121T001506, start_date=20241121T001508, end_date=20241121T001508
[2024-11-21T00:15:08.958+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-21T00:15:08.970+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-21T00:15:08.972+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
